modules/openapi-generator/src/main/resources/python/python-experimental/api_client.mustache

```diff
@@ -699,23 +699,38 @@ class ApiClient(object):
 
         return ss
 
-    def get_authorization_header(self, hdrs, signed_msg):
+    def get_authorization_header(self, signed_headers, signed_msg, created_ts):
         """
         Calculates and returns the value of the 'Authorization' header when signing HTTP requests.
         
-        :param hdrs : The list of signed HTTP Headers
-        :param signed_msg: Signed Digest
+        :param signed_headers : The list of signed HTTP Headers.
```

Comment:
Can you include the types of the list? Should it be list of str?

Reply:
Changing to:
```A list of strings. Each value is the name of a HTTP header that must be included in the HTTP signature calculation